components/automate-ui/src/app/modules/token/token-details/api-token-details.component.spec.ts

```diff
@@ -162,6 +149,34 @@ describe('ApiTokenDetailsComponent', () => {
     expect(component.updateForm.controls.projects.pristine).toEqual(true);
   });
 
+  using([
```

Comment:
These tests are identical to those for teams, modulo substituting a team object with a token object.